flang/lib/Optimizer/Transforms/RewriteLoop.cpp

```diff
@@ -68,22 +74,24 @@ class CfgLoopConv : public mlir::OpRewritePattern<fir::LoopOp> {
     rewriter.setInsertionPointToEnd(initBlock);
     auto diff = rewriter.create<mlir::SubIOp>(loc, high, low);
     auto distance = rewriter.create<mlir::AddIOp>(loc, diff, step);
-    auto iters = rewriter.create<mlir::SignedDivIOp>(loc, distance, step);
+    mlir::Value iters =
```

Comment:
Is this auto --> mlir::Value change intentional?

Reply:
Yes.

Reply:
It seems out of character.  Is it required?

Reply:
Yes. We need to rewrite `iters` to be the value of a different operation when the option is thrown.

Reply:
ok; thanks for the explanation